events/api.py

```diff
@@ -1950,6 +1950,8 @@ def _filter_event_queryset(queryset, params, srs=None):
             search_vals.append(event_types[v])
 
         queryset = queryset.filter(type_id__in=search_vals)
+    else:
+        queryset = queryset.filter(type_id=1)
```

Comment:
Instead of hardcoded type_id value, a reference to `Event.Type_Id.GENERAL` should be used: `Event.objects.filter(type_id=Event.Type_Id.GENERAL)`, which makes this 

```
    else:
        queryset = queryset.filter(type_id=Event.Type_Id.GENERAL)

Reply:
absolutely!